.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,38 @@
+---
+name: User story
+about: Suggest an idea the agile way :-)
+title: ''
+labels: Type:Story
+assignees: ''
+
+---
+
+# Description
+
+## User Stories
+
+* > As a ..., I want to ... so that ... (please stick to who, what, why)
+
+## Value
+
+## Acceptance Criteria
+
+## Definition of ready
+[ ] everybody needs to understand the value written in the user story
```

Comment:
In general, why do we start, the sentences lower case?

Reply:
I just copied over the story template from jira... adapted it a little bit now, please check again